### PR TITLE
Data

### PR DESCRIPTION
**Pull request by jGleitz**
Added information about the data Beagle deals with. Not too sure about some things, though. So I’d love to hear your comments.

This pull requests depends on #99 for logical reasons. Do not merge before #99!

### CODE REVIEW

**Comment by jGleitz** [comment]
[View this PR’s rendered PDF](http://jgleitz.github.io/Beagle/branches/data/Requirements%20Specification.pdf)

**Comment by acBerger** [comment]
lgtm

File: doc/Requirements Specification/Parts/Models.lyx [hunk 1]
```diff
@@ -120,8 +120,20 @@ Scenario
...
+\begin_layout Plain Layout
+all the people, living life in peace! (You-hu) You may say I'm a dreamer,
+ but I'm not the only one.
+ I hope one day you'll join us and the world will be as one.
+\end_layout
+
+\end_inset
+
+, that a Java based online shop is running on a middle-class web server
+ of a company named
```
Comment (flyfl) [inline doc/Requirements Specification/Parts/Models.lyx]: That's delicious!

**Comment by viridispirus** [comment]
lgtm

**Pull request merged**